examples/expandable.py

```diff
@@ -41,8 +41,8 @@ class MainWindow(SplitApplicationWindow):
     def _create_lhs(self, parent):
         """ Creates the left hand side or top depending on the split. """
 
-        self._expandable = expandable = ExpandablePanel(parent)
-        self._expandable.create()
+        self._expandable = expandable = ExpandablePanel()
+        self._expandable.create(parent)
```

Comment:
Nitpick level, but this and similar uses do read a bit oddly to me - it reads as though we're creating the parent. `create(parent=parent)` might be less susceptible to misinterpretation.

Reply:
I think I disagree on this one, since the parent is being passed through in a `_create_*` method, so the context is clear.  But I was in two minds about whether to even change this example because it not a very realistic example...